adapter_files/precice.cpp

```diff
@@ -182,17 +187,17 @@ double Precice::initialize() {
     displDeltaID = new int[globalNumberWetSurfaces];
     for (int i = 0; i < globalNumberWetSurfaces; i++) {
       // Get preCICE meshIDs
-      meshID[i] = solverInterface.getMeshID("SU2_Mesh" + to_string(i));
+      meshID[i] = solverInterface.getMeshID(preciceMeshName + (i == 0 ? "" : to_string(i)));
```

Comment:
With this we still require that the meshes are named as `Fluid-Mesh` (no `0`), `Fluid-Mesh1`, `Fluid-Mesh2`, ... right?

Reply:
No, number `0` is an empty string. So that we have  `Mesh`, `Mesh1`...

Reply:
I think we wrote exactly the same thing. :sweat_smile: 

Do we want this like this? If a user wants a simulation with one mesh, they name it e.g. `Mesh`. But if they want two meshes, they have to name them `Mesh` and `Mesh1`. This is not ideal, but I guess it is already good enough, as it simplifies the single mesh cases.

Reply:
In my opinion, the difference is that the user needs to spend some thoughts about different names of the coupling mesh in case he has more than one. So, it is more expected than in the case of only one interface.

Reply:
> I think we wrote exactly the same thing. sweat_smile 

now I get it :D